{"uid":"e5e2c59ed465afbd","name":"test_build_and_deploy","fullName":"tests.integration.ha_tests.test_replication#test_build_and_deploy","historyId":"39a877d256b35462b6ac3406bbac8930","time":{"start":1720661581548,"stop":1720661899417,"duration":317869},"description":"Build and deploy three unit of PostgreSQL.","descriptionHtml":"<p>Build and deploy three unit of PostgreSQL.</p>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/test_replication.py\", line 51, in test_build_and_deploy\n    await ops_test.model.wait_for_idle(status=\"active\", timeout=1500)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1720661581299,"stop":1720661581300,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1720661581300,"stop":1720661581546,"duration":246},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1720661581300,"stop":1720661581300,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1720661581547,"stop":1720661581547,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy three unit of PostgreSQL.","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/test_replication.py\", line 51, in test_build_and_deploy\n    await ops_test.model.wait_for_idle(status=\"active\", timeout=1500)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","steps":[],"attachments":[{"uid":"4845beb1b7cca7a","name":"log","source":"4845beb1b7cca7a.txt","type":"text/plain","size":3239}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1720661899451,"stop":1720661899452,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1720661940104,"stop":1720661940797,"duration":693},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1720661940798,"stop":1720661940799,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1720661940799,"stop":1720661940799,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1720661940799,"stop":1720661940800,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_replication"},{"name":"host","value":"large-430-1c3e787fbdb09791ff9f9306"},{"name":"thread","value":"11010-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"69994038ea1f3c01","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":32,"unknown":1,"total":34},"items":[{"uid":"2a8069afee228d56","reportUrl":"../104//#testresult/2a8069afee228d56","status":"passed","time":{"start":1720575363160,"stop":1720576035712,"duration":672552}},{"uid":"39d1db3bb688906c","reportUrl":"../103//#testresult/39d1db3bb688906c","status":"passed","time":{"start":1720488792537,"stop":1720489497256,"duration":704719}},{"uid":"b8783e83d761117","reportUrl":"../102//#testresult/b8783e83d761117","status":"passed","time":{"start":1720402235662,"stop":1720402973828,"duration":738166}},{"uid":"c4c9ec70cff54cf5","reportUrl":"../101//#testresult/c4c9ec70cff54cf5","status":"passed","time":{"start":1720315988270,"stop":1720316668367,"duration":680097}},{"uid":"ed7b23314779c487","reportUrl":"../100//#testresult/ed7b23314779c487","status":"passed","time":{"start":1720229813818,"stop":1720230529000,"duration":715182}},{"uid":"164aadb3290904c","reportUrl":"../99//#testresult/164aadb3290904c","status":"passed","time":{"start":1720143064671,"stop":1720143802655,"duration":737984}},{"uid":"32aa3b749aed7f01","reportUrl":"../98//#testresult/32aa3b749aed7f01","status":"unknown","time":{}},{"uid":"186380104bb4244c","reportUrl":"../97//#testresult/186380104bb4244c","status":"passed","time":{"start":1719970096846,"stop":1719970824965,"duration":728119}},{"uid":"96d0e7db9560cd32","reportUrl":"../96//#testresult/96d0e7db9560cd32","status":"passed","time":{"start":1719883806368,"stop":1719884540822,"duration":734454}},{"uid":"e6164f60dfced317","reportUrl":"../95//#testresult/e6164f60dfced317","status":"passed","time":{"start":1719797436216,"stop":1719798212985,"duration":776769}},{"uid":"469cc42923b2361d","reportUrl":"../94//#testresult/469cc42923b2361d","status":"passed","time":{"start":1719710999782,"stop":1719711647357,"duration":647575}},{"uid":"d16c845f4bdb0a49","reportUrl":"../93//#testresult/d16c845f4bdb0a49","status":"passed","time":{"start":1719624720385,"stop":1719625466491,"duration":746106}},{"uid":"d8bd5a3b0e8885b9","reportUrl":"../92//#testresult/d8bd5a3b0e8885b9","status":"passed","time":{"start":1719538244659,"stop":1719538967376,"duration":722717}},{"uid":"5c764a86e2f15324","reportUrl":"../91//#testresult/5c764a86e2f15324","status":"passed","time":{"start":1719451948786,"stop":1719452729407,"duration":780621}},{"uid":"eefa1e89c0654c4c","reportUrl":"../88//#testresult/eefa1e89c0654c4c","status":"passed","time":{"start":1719365302352,"stop":1719366341151,"duration":1038799}},{"uid":"3e08cff05bbd4933","reportUrl":"../87//#testresult/3e08cff05bbd4933","status":"passed","time":{"start":1719278875892,"stop":1719279895052,"duration":1019160}},{"uid":"5fb76ac7fa3eb6c2","reportUrl":"../86//#testresult/5fb76ac7fa3eb6c2","status":"passed","time":{"start":1719192511968,"stop":1719193428808,"duration":916840}},{"uid":"d59a8b38cb61e06b","reportUrl":"../85//#testresult/d59a8b38cb61e06b","status":"passed","time":{"start":1719106154945,"stop":1719107093075,"duration":938130}},{"uid":"e7894ccb3c8ba449","reportUrl":"../84//#testresult/e7894ccb3c8ba449","status":"passed","time":{"start":1719019662782,"stop":1719020740488,"duration":1077706}},{"uid":"44071ea9554be105","reportUrl":"../83//#testresult/44071ea9554be105","status":"passed","time":{"start":1718933265891,"stop":1718934242030,"duration":976139}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"e5e2c59ed465afbd.json","parameterValues":[]}